pyproject.toml

```
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "mxcubecore"
-version = "1.76.0"
+version = "1.77.0"
```

As far as I understood the process, that should not be done from within a pull request.

```suggestion
version = "1.76.0"
```

---

Its indeed strange that the PR includes a previous commit, from the CI (#4f6c5d9)

---

That happens if I get the latest updated from the upstream develop to agruzinov/develop through github web interface.